tests/testthat/test-fleet-interface.R

```diff
@@ -21,42 +20,42 @@ fleet module", {
   # Expect code produces error when ID of selectivity is a character string
   expect_error(fleet1$SetSelectivity("id"))
 
-  fims$clear()
+  clear()
 })
 
 
 test_that("Fleet: SetAgeCompLikelihood works", {
   fims <- Rcpp::Module("fims", PACKAGE = "FIMS")
```

Comment:
Should these fims module loads on lines 28, 37, 46, and 55 be deleted, I'm not sure if they are used and you have deleted them in other places?